src/pages/explore/[[...slug]].js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+import ExplorePage from "@/pesayetu/components/ExplorePage";
+import Page from "@/pesayetu/components/Page";
+import getPostTypeStaticProps from "@/pesayetu/functions/postTypes/getPostTypeStaticProps";
+import fetcher from "@/pesayetu/utils/fetcher";
+
+export default function Explore(props) {
+  return (
+    <Page {...props}>
+      <ExplorePage {...props} />
+    </Page>
+  );
+}
+
+export async function getStaticPaths() {
+  const result = await fetcher(
+    `https://ng.hurumap.org/api/v1/all_details/profile/1/geography/KE/?format=json`
+  );
+  const paths = result?.children?.county?.features?.map(
+    ({ properties: { code, level } }) => {
+      const s = `${level}-${code}`;
+      return {
+        params: {
+          slug: [s],
+        },
+      };
+    }
+  );
+
+  return {
+    paths,
+    fallback: "blocking",
```

Comment:
Why?

Reply:
@kilemensi copy & paste..  😆  we don't need this